infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -56,16 +60,50 @@ def run_shell_command(command, env_vars):
 ###################
 
 
-def make_backup_fpath():
-    return BACKUP_DIR / datetime.now().strftime(FILENAME_DATETIME_FORMAT)
+def make_object_name():
+    return datetime.now().strftime(FILENAME_DATETIME_FORMAT)
 
 
-def backup_database(db_backup_gzip_fpath):
+def make_backup_fpath(object_name):
+    return BACKUP_DIR / object_name
+
+
+def create_s3_client():
```

Comment:
Love this. Needs a return type. Here it is 
![image](https://user-images.githubusercontent.com/32227767/154885666-557e4018-44e8-4f06-bd8c-6a7937fe9ea6.png)

Reply:
But seriously, the way you set this up using the `boto3.session` is fantastic.